app/src/main/java/org/p2p/wallet/bridge/model/BridgeFee.kt

```diff
@@ -18,8 +19,10 @@ data class BridgeFee(
     val name: String,
     val decimals: Int
 ) : Parcelable {
-    val amountInToken
-        get() = amount?.toBigDecimal()?.orZero()?.divide(decimals.toPowerValue()).orZero()
+    val amountInToken: BigDecimal
+        get() = amount?.toBigDecimal()
+            .orZero()
```

Comment:
zero division

Reply:
you forgot to transform from lamports. 

`amount.toBigIntegerOrZero().fromLamports(decimals)`

Reply:
sorry, not my code, won't touch the division and stuff

Reply:
reverted